webpack.config.js

```diff
@@ -0,0 +1,56 @@
+const merge = require('webpack-merge');
+const path = require('path');
+const fs = require('fs');
+const glob = require('glob');
+const _ = require('lodash');
+
+/* eslint-disable import/no-unresolved, import/no-extraneous-dependencies */
+const { config } = require('@ovh-ux/manager-webpack-config')({
+  template: './client/index.html',
+  basePath: './client',
+  lessPath: [
+    './client/app',
+    './client/components',
+    './node_modules',
+  ],
+  root: path.resolve(__dirname, './client/app'),
+  assets: {
+    files: [
+      { from: path.resolve(__dirname, './client/app/common/assets'), to: 'assets' },
+      { from: path.resolve(__dirname, './node_modules/angular-i18n'), to: 'angular-i18n' },
+      { from: path.resolve(__dirname, './client/**/*.html'), context: 'client' },
+    ],
+  },
+});
+/* eslint-enable */
+
+const folder = './client/app/telecom';
+const bundles = {};
+
+fs.readdirSync(folder).forEach((file) => {
+  const stats = fs.lstatSync(`${folder}/${file}`);
+  if (stats.isDirectory()) {
+    const jsFiles = glob.sync(`${folder}/${file}/**/*.js`);
+    if (jsFiles.length > 0) {
+      bundles[file] = jsFiles;
+    }
+  }
+});
+
+module.exports = merge(config, {
```

Comment:
Can we serve this over `https`?

Reply:
We already discuss about this, yes we can :)

We have disable it because we don't use HTTP2 atm.

We should enable HTTP2/HTTPS in another pull request, if it's ok for you :)